### ISSUE TITLE
about normalize

### ISSUE THREAD

**Issue by Tmn07** [ZZR0/ISSTA21-JIT-DP#1]
hi, i find in the function baseline_algorithm in ./JIT_Baseline/baseline.py
you scale X_train, X_test respectively. Maybe it would be more appropriate to normalize the test set using the mean, variance of the training set, like this
``` 
  scaler = preprocessing.StandardScaler()
  X_train = scaler.fit_transform(X_train)
  X_test = scaler.transform(X_test)
```

i had use this normalize and run RQ4_T8 experiment, there are some changes in result, such as 
```
|    project:jdt model:lr type:WP   |
['Train Time: 0.023188 s\n', 'Eval Time: 0.000293 s\n', 'Accuracy: 0.625000 -- Precision: 0.606232 -- Recall: 0.666667 -- F1: 0.000000 -- AUC: 0.680206\n']

|    project:platform model:lr type:WP   |
['Train Time: 0.082072 s\n', 'Eval Time: 0.000384 s\n', 'Accuracy: 0.719982 -- Precision: 0.591667 -- Recall: 0.402266 -- F1: 0.000000 -- AUC: 0.757886\n']
```

Finally, thanks to your great work

**Comment by ZZR0** [comment]
Thanks for your interest and the ideas you provided. We are sorry that we did not perform a complete normalize experiment. For the sake of fairness, we have simply followed the normalization scheme of previous work [DeepJIT](https://github.com/hvdthong/Just-in-time-defect-prediction/blob/master/baseline_JIT/load_data_yasu.py) . Thank you for your efforts on this issue and we hope to see more of your findings.
